documentation/docs/introduction.md

```diff
@@ -19,47 +19,56 @@ In this guide, we will go through the most important concepts that developers wi
 
 ## Overview
 
-### Chapter 2: Decentralized Identity
+### Chapter 2: [Decentralized Identity](./decentralized_identity)
```

Comment:
Is there a reason this starts at "Chapter 2" instead of "Chapter 1"? The Introduction page is basically a table of contents, not a chapter in itself.